ts/src/okx.ts

```diff
@@ -874,8 +880,8 @@ export default class okx extends Exchange {
                     'ALGO': 'Algorand',
                     'BHP': 'BHP',
                     'APT': 'Aptos',
-                    'ARBONE': 'Arbitrum one',
-                    'AVAXC': 'Avalanche C-Chain',
+                    'ARBONE': 'Arbitrum One',
+                    'AVAXC': 'Avalanche C',
```

Comment:
@NeOMakinG I think we should update our mappings for networks here. Could you please check it?

Reply:
@neeeekitos We don't use this internal mapping for OKX, we already have `Arbitrum One` inside our mapping, it seems like it works as it should